src/Cell.cpp

```diff
@@ -1137,12 +1137,29 @@ void Cell::ringify(std::vector<Cell*>& subcells, double max_radius) {
   std::vector<ZCylinder*>::iterator iter2;
   std::vector<Cell*>::iterator iter3;
 
-  /* Compute the area to fill with each equal volume ring */
-  double area = M_PI * fabs(radius1*radius1 - radius2*radius2) / _num_rings;
+  /* Compute the increment, either by radius or area, to use to construct
+   * the concentric rings */
+  double increment;
+
+  /* If there is no outer bounding surface, make the rings have the same
+  * radius increment. */
+  if (halfspace1 == 0)
+    increment = fabs(radius1 - radius2) / _num_rings;
+
+  /* If there is an outer bounding surface, make the rings have the same
```

Comment:
Might clarify this comment with something like "(e.g., fuel in a pin cell universe)"